lib/decidim/spam_detection/block_spam_user_command.rb

```diff
@@ -65,9 +66,22 @@ def reason
         I18n.t("blocked_user.reason", probability: @probability)
       end
 
+      def generate_nickname
+        max_attempts = 10
+
+        max_attempts.times do
+          random_key = SecureRandom.hex(5)
```

Comment:
What is the maximum length for nickname ? Can we increase the size of the random_key to prevent collision ?